src/main/java/raisetech/studentmanagement/controller/StudentController.java

```diff
@@ -2,16 +2,17 @@
 
 import java.util.List;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.RequestParam;
-import org.springframework.web.bind.annotation.RestController;
 import raisetech.studentmanagement.controller.converter.StudentConverter;
 import raisetech.studentmanagement.data.Student;
 import raisetech.studentmanagement.data.StudentCourse;
 import raisetech.studentmanagement.domain.StudentDetail;
 import raisetech.studentmanagement.service.StudentService;
 
-@RestController
+@Controller//@RestControllerから変更

```

Comment:
変更履歴をコメントで残す必要はないですね。
```suggestion
@Controller

```

Reply:
もし学習目的でコメントを残すということであればいまのままでもよいです！

Reply:
何をやっているのか分からなくなりそうなので、コメント書いてました！
学習目的で残しておこうと思います！
ありがとうございました！